src/sources/hydra.rs

```diff
@@ -146,14 +147,39 @@ impl Worker {
     }
 }
 
+fn intersect_from_config(intersect: &IntersectConfig) -> Point {
+    match intersect {
+        IntersectConfig::Origin => {
+            info!("intersecting origin");
+            Point::Origin
+        }
+        IntersectConfig::Tip => {
+            panic!("intersecting tip not currently supported with hydra as source")
+        }
+        IntersectConfig::Point(slot, hash_str) => {
+            info!("intersecting specific points");
```

Comment:
```suggestion
            info!("intersecting specific point");
```
was copied from `sources/n2c.rs`